lib/autorestPlugin/pluginHost.js

```diff
@@ -0,0 +1,204 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
for code that is all about logic and not about protocol, you can use this: https://github.com/olydis/autorest-extension-base (see README, it's just 3 lines of overhead)